common/scala/src/main/resources/reference.conf

```diff
@@ -3,4 +3,5 @@ whisk.spi{
   MessagingProvider = whisk.connector.kafka.KafkaMessagingProvider
   ContainerFactoryProvider = whisk.core.containerpool.docker.DockerContainerFactoryProvider
   LogStoreProvider = whisk.core.containerpool.logging.DockerToActivationLogStoreProvider
+  LoadBalancerProvider = whisk.core.loadBalancer.LoadBalancerService
```

Comment:
I don't have a strong opinion, but it may be good to have more specific names to the Spi impl, e.g. `whisk.core.loadBalancer.DefaultLoadBalancerProvider' (not sure of a great name here; maybe KafkaLoadBalancerProvider?)

Reply:
Hmm I don’t like kafka in the name. The lb should be using an abstract messaging interface not specifically Kafka’s. ContainerPoolLoadBalancer? We can pull the renaming into a seperate issue and come up with a better name. I agree the choice of LoadBalancerService may not have been the best (it dates back to when we had a seperate microservice to LB).